src/Alias/Finder.cs

```diff
@@ -58,4 +58,39 @@ public static class Finder
             }
         }
     }
+    
+    public static IEnumerable<string> FilterAssemblies(
```

Comment:
seems this is mostly a copy of FindAssemblyInfos

can some of the code be shared?